### DISCUSSION TITLE
Include `libexec/zellij-utils/assets` directory in homebrew assets

### DISCUSSION THREAD

**Discussion by blogabe** [zellij-org/zellij#3094] [Ideas]
New to Zellij on macOS. I went looking for aliases, eg, `ze` for `zellij edit` and could not find them. I didn't find the aliases I was looking for and created my own. I'm curious though if it's possible to include the `zellij-utils/assets` directory when homebrew installs as a bottle or through source.

**Comment by imsnif** [comment]
Hey, we do not control the homebrew installation. I'd try to talk to them...

**Reply by blogabe** [reply]
Ok thank you. Will follow back on this thread after I hear back from homebrew. Feel free to close the conversation if you prefer.
